src/duckstation-qt/memoryscannerwindow.cpp

```diff
@@ -90,6 +90,10 @@ MemoryScannerWindow::MemoryScannerWindow() : QWidget()
 {
   m_ui.setupUi(this);
   connectUi();
+
+  char tempvar[256];
+  sprintf(tempvar, "Address of RAM for HxD Usage: 0x%p", Bus::g_ram);
+  m_ui.cheatEngineAddress->setText(tempvar);
```

Comment:
Please don't use sprintf, it's unsafe (even if it's not going to overflow).

```suggestion
  m_ui.cheatEngineAddress->setText(tr("Address of RAM for HxD Usage: 0x%1").arg(reinterpret_cast<qulonglong>(Bus::g_unprotected_ram), 16, 16, QChar('0')));
```